src/app/features/task-list/task-list.component.ts

```diff
@@ -30,6 +33,13 @@ export class TaskListComponent implements OnInit, OnDestroy {
         const tasks = response.data.tasks;
         this.taskService.taskSubject.next(tasks);
       });
+
+    this.tasks$ = this.route.paramMap.pipe(
```

Comment:
this doesnt make sense

Reply:
you already assigned taskService.tasks$ to the tasks$ observable but now youre reassigning it